Declarations/Contracts/Constants.ts

```diff
@@ -18,4 +18,18 @@ export function domainSupportsProperties(domain: Domain): domain is ISupportProp
         domain instanceof PageViewData ||
         domain instanceof RemoteDependencyData ||
         domain instanceof RequestData;
-}
\ No newline at end of file
+}
+
+/**
+ * Subset of Connection String fields which this SDK can parse. Lower-typecased to
+ * allow for case-insensitivity across field names
+ * @interface IConnectionStringFields
+ */
+export interface IConnectionStringFields {
+    authorization?: string;
+    instrumentationkey?: string;
+    ingestionendpoint?: string;
+    liveendpoint?: string;
+
+    [key: string]: string; // catch all for non-recognized fields
```

Comment:
I don't like including this here. Ideally there is a better way to do case-insensitive parsing of the string without having to check which interface member each connection string member potentially belongs to